src/doc/src/reference/resolver.md

```diff
@@ -12,6 +12,25 @@ resolver.
 [dependency specification]: specifying-dependencies.md
 [`cargo tree`]: ../commands/cargo-tree.md
 
+## `Cargo.lock` lock-files
+
+The `Cargo.lock` file provides deterministic builds at different times and on
```

Comment:
There are some other places the book mentions Cargo.lock

* https://doc.rust-lang.org/nightly/cargo/appendix/glossary.html
* https://doc.rust-lang.org/nightly/cargo/guide/cargo-toml-vs-cargo-lock.html#cargotoml-vs-cargolock
* https://doc.rust-lang.org/nightly/cargo/faq.html#why-have-cargolock-in-version-control
* The previous paragraph.

it is a bit duplicate everywhere. We might want some level of consolidation.